packages/core/src/modules/dids/repository/DidRecord.ts

```diff
@@ -23,6 +23,7 @@ export interface DidRecordProps {
 
 export interface CustomDidTags extends TagsBase {
   recipientKeyFingerprints?: string[]
+  alsoKnownAs?: string[]
```

Comment:
We can extract this from the did document itself right? Or are they set manually?

Reply:
At the moment they are set manually, much in the same way that `recipientKeyFingerprints` (and actually will be used similarly once we support DIDComm V2, where we'll use something like `did.findByDid` instead of `did.findByRecipientKey`).

They can be also extracted from the did document, but in cases like did:peer:2 and did:peer:4 we are not storing the whole DID Document in `didDocument` property.

I'm thinking that maybe this general approach of using `alsoKnownAs` is not a good idea, and that we can call it differently (something like `alternateDids`) to make it clear that it is a particular case of certain dids (currently did:peer:4).